### DISCUSSION TITLE
mypy check failed for URL/Secret typing from Config

### DISCUSSION THREAD

**Discussion by chuckliu1979** [encode/starlette#1587] [Q&A] [answered]
for this code :

from starlette.config import Config
from starlette.datastructures import URL, Secret

config = Config(".env")

URI: URL = config("URI", cast=URL, default="")
PASSWORD: Secret = config("PASSWORD", cast=Secret, default="")

when run mypy check, it failed:
error: Incompatible types in assignment (expression has type "object", variable has type "URL")  [assignment]
error: Incompatible types in assignment (expression has type "object", variable has type "Secret")  [assignment]

**Comment by Kludex** [comment]
Is this still an issue? We've made some typing changes over there.

**Comment by Kludex** [comment, marked as answer]
Closing this as outdated.
